Tasks/Salanenka/Task7/Task7.py

```diff
@@ -0,0 +1,23 @@
+def summa (a):
+    sum=0
+    for i in a:
+        if type(i) == int:
+            sum+=i
+        if type(i) == list:
+            sum+=summa (i)
+    return sum
+
+d = [1, 2, [2, 4, [[7, 8], 4, 6]]]
+print (summa(d))
+
+def fib(n):
+    def fi (n):
+        if n<=1: 
+            return n
+        return (fi(n-1)+fi(n-2))
```

Comment:
Hi!) It's realy difficult to understand fo me! =) I debuged it for almost 20 times before catch the logic.)) Cool solution!)